modules/couchbase/charts.go

```diff
@@ -0,0 +1,76 @@
+package couchbase
+
+import (
+	"github.com/netdata/go.d.plugin/agent/module"
+)
+
+type (
+	Charts = module.Charts
+	Chart  = module.Chart
+	Dim    = module.Dim
+)
+
+var dbPercentCharts = Chart{
+	ID:    "couchbase_quota_percent_used_stats",
+	Title: "Quota Percent Used Per Bucket",
+	Units: "%",
+	Fam:   "quota percent used",
+	Ctx:   "couchbase.couchbase_quota_percent_used_stats",
```

Comment:
lets make it `couchbase.couchbase_quota_percent_used_stats` => `couchbase.quota_percent_used_stats` (revmoe `couchbase` after `.`)

Please do it for all contextes